### DISCUSSION TITLE
Telescope steering calibration

### DISCUSSION THREAD

**Discussion by StuartCT** [Stellarium/stellarium#1588] [Q&A]
I've got Stellarium steering my telescope, but it's a little out. I'm currently taking subs of M51, but I had to manually adjust the scope a little from where Stellarium  put it. (you can see from the capture that it's a little to the east of where it thinks it is)

1. is there a way to tell Stellarium to make a little adjustment to its alignment?

2. can I adjust the scope slew from within Stellarium? (as it's a bit tedious going out and tweaking the hand controller, taking a photo to see where it is, going out to tweak again etc etc etc)
![Stellarium jpg 14f3750d0296696f7f8c6328d15990b4](https://user-images.githubusercontent.com/34127167/113557490-0008c400-95f6-11eb-8c20-7e284a308325.jpg)

**Comment by alex-w** [comment]
Did you tried use SYNC command?

**Reply by StuartCT** [reply]
No, I did not. What does that do? How does it know what to sync with?

**Reply by andyhellon** [reply]
I'm running  a meade lx90 on an ascom meade generic driver and I have the same problem. using sync command didn't work.  Telescope direction control from within stellarium would be a good addition

**Reply by howshore** [reply]
I see small position discrepencies on my system (Stellarium 0.21.2 and Syncscan version 3.39.10) as well.  I assume it is because the PC and the mount disagree about the time.  I set the mount's time from UTC as displayed by a mobile telephone's GPS receiver.  My PC gets its time from Windows default network time server.  It would be useful for the telescope plugin to be able to set Stellarium's idea of the time from the mount.  I start the mount and set its time, then calibrate the pointing with three stars, and finally start the PC.  If Stellarium changed the mount's time then I'd need to do another three star calibration.
I don't know of a SYNC command either?
I manually point the telescope to a bright star near my object of interest, then measure the offset Stellarium shows.  I then search Stellarium's screen for a star with the opposite offset from my object of interest and ask Stellarium to slew to that star.  To maintain the tracking I use the program PhD operating the mount through the tracking port and a finder scope with a camera.  Over a few hours Stellarium shows the telescope's position slowly drifting although PhD keeps the mount truly locked to the sky.  My mount isn't exactly aligned of course, and the location can be specified only in degrees and minutes (around +/-900m), but the biggest reason for drift is probably that Stellarium can't account for the real atmosphere.  Stellarium has to assume that light travels in straight lines ( with a standard atmospheric model? ), PhD does not.

**Reply by gzotti** [reply]
Stellarium sends "GOTO RA/DEC" commands. The mount has to know where that is. I.e., align the mount using the mount's controls, then it should be fine.
